README.md

```diff
@@ -16,3 +20,53 @@ Dev:
 - Nodemon: auto restarts server
 - Jest
 - Supertest
+
+## API routes
+
+| Endpoint          | Functionality            | Deployed         |
```

Comment:
Awesome that you added the table and examples! Super clear

Reply:
Yeah it's cool isn't it?  Handy table Taran pulled up